cpp/src/parquet/encoding.cc

```diff
@@ -3417,7 +3417,8 @@ std::unique_ptr<Encoder> MakeEncoder(Type::type type_num, Encoding::type encodin
 }
 
 std::unique_ptr<Decoder> MakeDecoder(Type::type type_num, Encoding::type encoding,
-                                     const ColumnDescriptor* descr) {
+                                     const ColumnDescriptor* descr,
+                                     ::arrow::MemoryPool* pool) {
```

Comment:
Plain decoder does not need a pool?

Reply:
Some decoders, like `DeltaBitPacked` may requires a pool, so I added. Currently, Plain doesn't need a Pool